components/elm/src/external_models/emi/src/elm_stub/elm_types_stub/CNCarbonStateType.F90

```diff
@@ -16,7 +16,7 @@ module CNCarbonStateType
   use clm_varctl             , only : nu_com, use_fates, use_crop
 
   ! bgc interface & pflotran
-  use clm_varctl             , only : use_clm_interface, use_pflotran, pf_cmode
+  use clm_varctl             , only : use_elm_interface, use_pflotran, pf_cmode
```

Comment:
Should clm_varctl be renamed?

Reply:
The code in `components/elm/src/external_models/emi/src/elm_stub` is for the stub case and is not useful any longer. I will remove it in a future PR.